src/main/webapp/app/exam/participate/exercises/quiz/quiz-exam-submission.component.ts

```diff
@@ -47,7 +47,18 @@ export class QuizExamSubmissionComponent extends ExamSubmissionComponent impleme
     @Input()
     studentSubmission: AbstractQuizSubmission;
 
-    @Input() exercise: QuizExercise;
+    @Input() exerciseId: number | undefined;
```

Comment:
I don't like to have so many `undefined` inputs here. Does the page really work if those values are all undefined?